plugin/db/mysql/mysql.go

```diff
@@ -41,6 +41,7 @@ type Driver struct {
 func newDriver(dc db.DriverConfig) db.Driver {
 	return &Driver{
 		resourceDir: dc.ResourceDir,
+		binlogDir:   dc.BinlogDir,
```

Comment:
Should this be backupDir, as the driver will handle both binlog and backup files?

Reply:
Not really.

The backup dir is dataDir/backup/db/id/, and binlog dir is dataDir/backup/instance/id/.

Reply:
We may later refactor to associate api.Instance/api.Database with the Driver, then we only need one dataDir field, and the Driver could generate other directories itself.

Reply:
ok